src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -309,7 +309,7 @@ static inline void cavs_pm_runtime_core_en_memory(uint32_t index)
 static inline void cavs_pm_runtime_dis_dsp_pg(uint32_t index)
 {
 #if CAVS_VERSION >= CAVS_VERSION_1_8
-	uint32_t lps_ctl;
+	uint32_t lps_ctl, tries = 32, delay_count = 256;
```

Comment:
Can we make 'tries' a macro with a comment, that way it's obvious if anyone has to tune this later on. Delay count can be macro or const too.

Reply:
ok done